<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialEmitReaderWidget</class>
 <widget class="QWidget" name="SerialEmitReaderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lukulaite - Sarjaportti/USB - Monitori</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="textEdit"/>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="porttiBox">
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string> -&gt; </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="kirjoitusPorttiBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>Yhdist√§</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Sulje</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>SerialEmitReaderWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
